CODEOWNERS

```diff
@@ -1,9 +1,14 @@
+# Misc
```

Comment:
Github says "# Order is important; the last matching pattern takes the most
# precedence." . So we should define less specific codeowner paths before more specific codeowner paths. 

Example: with this change, now any changes made to `/core/chains/evm` should require @samsondav or @prashantkumar1982  instead of only @samsondav (previously, the `/core/chains` rule was overriding this behavior)

Reply:
Great catch, Augustus!
Can you put the github quote as top level comment in this file, so everyone knows to follow this always?